src/psyclone/tests/domain/gocean/transformations/gocean1p0_transformations_test.py

```diff
@@ -1527,6 +1529,12 @@ def test_ocl_apply(kernel_outputdir):
     psy, invoke = get_invoke("test11_different_iterates_over_"
                              "one_invoke.f90", API, idx=0, dist_mem=False)
     schedule = invoke.schedule
+    # Currently, moving the boundaries inside the kernel is a prerequisite
+    # for OCLTrans
```

Comment:
We need to extend the validation in OCLTrans to check that this has been done.

Reply:
My comments here were not accurate. It is in not a prerequisite of OCLTrans (the transformations can be applied in any order) so the validation would not help.

What it is at the moment is a prerequisite for the gen_ocl() done during the generation step, not doing it would already fail with a generation error that now mention this transformation, see gocean1p0_opencl_test.py::test_opencl_kernel_missing_boundary_symbol

Note that I expect this limitation to be short lived as hopefully #1134 will make this more flexible.